hack/run-operating-system-manager.sh

```diff
@@ -16,8 +16,14 @@
 
 set -e
 
-NAMESPACE="${NAMESPACE:-kube-system}"
```

Comment:
Let's stick with `NAMESPACE` being picked from the env

Reply:
All the consumers of this services are creating the secret in the `cloud-init-settings` namespace. Actually that namespace isn't created by OSM, it is created by KKP and should be created by KubeOne(once OSM is integrated there). Also OSM should have a very particular name space with a very limited auth scope, due to the fact that we are exporting the token and use. Thus it should be isolated from the whole cluster.

Reply:
Makes sense (y)